src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -174,7 +174,7 @@ AliasesToExport = 'Get-AzStorageContainerAcl', 'Start-CopyAzureStorageBlob',
                'Stop-CopyAzureStorageBlob', 'Enable-AzStorageSoftDelete', 
                'Disable-AzStorageSoftDelete', 'New-AzDatalakeGen2FileSystem', 
                'Remove-AzDatalakeGen2FileSystem', 'Get-AzDatalakeGen2FileSystem', 
-               'New-AzDataLakeGen2ItemAclObject'
+               'New-AzDataLakeGen2ItemAclObject', 'Invoke-AzureRmStorageAccountFailover'
```

Comment:
The name of cmdlet should be `Invoke-AzStorageAccountFailover`

Reply:
This alias is added, because the "Enable-AzureRmAlias" can't enabled alias for it.
This is because "Enable-AzureRmAlias" only works for the cmdlets in stable release, and the 'Invoke-AzureRmStorageAccountFailover' cmdlet is added when we still use this cmdlet format to preview module. So to make the cmdlet works with both old and new name, we make the old name as cmdlet alias.

Reply:
I have removed 'Invoke-AzureRmStorageAccountFailover', and inform feature team, the old style cmdlet name 'Invoke-AzureRmStorageAccountFailover' will not be supported when the feature GA in PSH.